lib/index.d.ts

```diff
@@ -637,6 +637,11 @@ declare module "react-native-firebase" {
          */
         currentUser: User | null
 
+        /**
+         * Gets/Sets the language for the app instance
+         */
+        languageCode: string | null;
```

Comment:
cc @chrisbianca not sure how to do a getter/setter in TypeScript...? This works for the getter but not sure if it's moan when trying to set it.

Reply:
@Ehesp it is ok, `languageCode` is a read/write property, the implementation is hidden.